### PR TITLE
Replacing the old esm shadow blur with faster blur

### PR DESCRIPTION
**Pull request by mrieggeramzn**
Kawase blur replaces the old gaussian blur. Performance increase was 2x or higher at max softness. Quality was as good if not better than previous blur. 

Ran ASV tests ok
Tested amd and nvidia in Editor with various light types at different resolutions
Measured performance differences in Renderdoc

### Old method mid softness
old way 512 mid soft

### Old method max softness 
old way 512 max soft

### New method (2x faster than old method max softness)
kawase

Signed-off-by: mrieggeramzn <mriegger@amazon.com>

### CODE REVIEW

**Review by HogJonny-AMZN — COMMENTED** [review]
This looks like a great addition.

**Comment by mrieggeramzn** [comment]
> 
> 
> This looks like a great addition.

Thanks Jonny! Really appreciate it.

File: Gems/Atom/Feature/Common/Assets/Shaders/Math/GaussianFilterFloatVertical.azsl [hunk 1]
```diff
@@ -1,60 +0,0 @@
...
-void MainCS(uint3 dispatchId: SV_DispatchThreadID)
-{
-    const float3 inputSize = GetImageSize(FilterPassSrg::m_inputImage);
-    const float3 outputSize = GetImageSize(FilterPassSrg::m_outputImage);
-
-    const uint shadowmapIndex = GetShadowmapIndex(
-        FilterPassSrg::m_shadowmapIndexTable,
-        dispatchId,
-        inputSize.x);
-    // Early return if thread is outside of shadowmaps.
```
Comment (amzn-tommy) [inline Gems/Atom/Feature/Common/Assets/Shaders/Math/GaussianFilterFloatVertical.azsl:25]: It looks like the kawase blur doesn't have any of these early returns, even though it's using the same GetShadowmapIndex function. Is there a reason that's not necessary anymore?

**Review by amzn-tommy — APPROVED** [review]

**Review by invertednormal — APPROVED** [review]

File: Gems/Atom/Feature/Common/Assets/Shaders/Math/GaussianFilterFloatVertical.azsl (hunk 1 above)
Comment (mrieggeramzn) [reply Gems/Atom/Feature/Common/Assets/Shaders/Math/GaussianFilterFloatVertical.azsl:25]: Really good eye, thanks for noticing this! I've added the old checks back in.

**Pull request merged**